src/tbb/governor.cpp

```diff
@@ -132,11 +132,11 @@ void governor::one_time_init() {
 static std::uintptr_t get_stack_base(std::size_t stack_size) {
     // Stacks are growing top-down. Highest address is called "stack base",
     // and the lowest is "stack limit".
-#if USE_WINTHREAD
```

Comment:
It seems that this macro no longer defined by oneTBB build system (while classic TBB [does define it](https://github.com/oneapi-src/oneTBB/blob/eca91f16d7490a8abfdee652dadf457ec820cc37/build/windows.cl.inc#L119-L120))

Reply:
It is never used in oneTBB.